src/extension.ts

```diff
@@ -78,6 +66,31 @@ function useStylefmt(document: vscode.TextDocument, range: vscode.Range): Promis
 		}));
 }
 
+/**
+ * Resolve Stylefmt config
+ */
+function useStylefmt(document: vscode.TextDocument, range: vscode.Range): Promise<IResult> {
+	const settings: IStylefmtOptions = vscode.workspace.getConfiguration().get('stylefmt');
+	const configResolver = new ConfigResolver(vscode.workspace.rootPath);
+	const options = {
+		packageProp: 'stylelint',
+		configFiles: [
+			'.stylelintrc',
+			'stylelint.config.js'
+		],
+		editorSettings: settings.config
+	};
+
+	return configResolver.scan(document.uri.fsPath, options).then((resolved) => {
```

Comment:
Here we should merge resolved config and config from `stylelint.configOverrides` option if `stylefmt.useStylelintConfigOverrides` option is enabled.